docs/source/develop/api/permissions.rst

```diff
@@ -1172,7 +1172,7 @@ Allows getting any `transaction <../../before_start/glossary.html#transaction>`_
 .. literalinclude:: ../../../../example/python/permissions/can_get_all_txs.py
     :language: python
     :linenos:
-    :lines: 11-58
+    :lines: 12-59
```

Comment:
maybe we should add a comment to that python source that these line numbers have to be kept matching it?

Reply:
Boundaries are defined here https://github.com/hyperledger/iroha/blob/e8dc37b4244d6332e4fb3967e54e989d7e6870d5/docs/permissions_compiler/rst.py#L111-L137